### DISCUSSION TITLE
fooocus wont start ?

### DISCUSSION THREAD

**Discussion by muaaz0707** [lllyasviel/Fooocus#3676] [Q&A]
![Untitled](https://github.com/user-attachments/assets/21e1bb6d-5a91-4ad3-8f93-3b9986d3ddb5)
 everthing don its says to make a public link share=true in lauch() and then the browser opens but the page wont load and in cmd it sayd focus paused press any key to continue and idea how to make it work????

**Comment by Rogala** [comment]
[Minimal Requirement](https://github.com/lllyasviel/Fooocus?tab=readme-ov-file#minimal-requirement)
